CMakeLists.txt

```diff
@@ -2,17 +2,23 @@ cmake_minimum_required(VERSION 3.9.2)
 
 cmake_policy(SET CMP0048 NEW)
 project(safe VERSION 1.0.0 LANGUAGES CXX)
+option(BUILD_TESTING "Build tests" ON)
```

Comment:
Is this wanted ?

Reply:
Yes it's just to make it explicit.

The equivalent of the option + enable_testing is to simply include(CTest)